src/LuzFaltex.Extensions.DependencyInjection/ServiceLookup/ServiceProviderCallSite.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace LuzFaltex.Extensions.DependencyInjection.ServiceLookup;
+
+internal sealed class ServiceProviderCallSite : ServiceCallSite
+{
+    public ServiceProviderCallSite() : base(ResultCache.None)
+    {
+    }
+
+    public override Type ServiceType { get; } = typeof(IServiceProvider);
+    public override Type ImplementationType { get; } = typeof(MutableServiceProvider);
```

Comment:
Only this line was changed from ``ServiceProvider`` -> ``MutableServiceProvider`` in this file.